osmtm/templates/task.history.mako

```diff
@@ -52,6 +52,8 @@ from osmtm.mako_filters import (
           <span><i class="glyphicon glyphicon-thumbs-down text-danger"></i> <b>${_('Invalidated')}</b> ${_('by')} ${user_link | n}</span>
         % elif step.state == step.state_validated:
           <span><i class="glyphicon glyphicon-thumbs-up text-success"></i> <b>${_('Validated')}</b> ${_('by')} ${user_link | n}</span>
+        % elif step.state == step.state_removed:
+          <span><i class="glyphicon icon-split"></i> <b>${_('Split')}</b> ${_('by')} ${user_link | n}</span>
```

Comment:
By reading the translation here I just figured out that this may not work in some languages.
This should be addressed in a different issue but it would be better to have something looking like:
```
${_('Split by ${link}', mapping={'link': user_link})}
```